src/main/java/uk/gov/hmcts/cmc/claimstore/services/pilotcourt/PilotCourtService.java

```diff
@@ -95,25 +116,43 @@ public boolean isPilotCourt(String courtName) {
 
         Map<String, PilotCourt> pilotCourts = new HashMap<>();
         while (iterator.hasNext()) {
-            CSVRecord next = iterator.next();
+            CSVRecord csvRecord = iterator.next();
+
+            assert (csvRecord.size() == PilotCourtCSVHeader.values().length) : String.format("Pilot court "
+                + "configuration for %s is expected to have %d values",
+                csvRecord.toString(),
+                PilotCourtCSVHeader.values().length);
```

Comment:
I doubt assertions are enabled in prod

Reply:
If they aren't that is a bad practice, unless the assertions cause a significant performance hit. However it shouldn't matter as the configuration it asserts is something that is added in development, not on an environmental level.

Reply:
Disagree re: bad practice, but either way this would be the only place in claim-store where it'd be used - no real value in deviating.